code/main.cpp

```diff
@@ -5,8 +5,8 @@ int main(int argc, char *argv[])
 {
     QApplication app(argc, argv);
     MainWindow *w = new MainWindow();
-
     w->show();
 
     return app.exec();
 }
+

```

Comment:
[Nit] Not sure if you meant to commit these changes too. Maybe just revert this? Otherwise it'll show that you last worked on this file in Github.